### DISCUSSION TITLE
是否考虑移除使用 findDOMNode

### DISCUSSION THREAD

**Discussion by KingAmo** [ant-design/ant-design-mobile#6139] [General]
控制台总是报错很难受：
```
 findDOMNode is deprecated in StrictMode.
````

**Comment by miracles1919** [comment]
目前看没有好的的替代方案... 

一般是尝试用 ref 替代 findDOMNode，但 ref 是可以自定义暴露字段的，例如 `Button` 组件，ref 得到的是 `{ nativeElement }` ，没法统一处理

**Reply by miracles1919** [reply]
关了 Strict Mode 就没有 warning 了 - -
